src/dags/financien_grootboek.py

```diff
@@ -0,0 +1,132 @@
+from airflow import DAG
+from airflow.operators.postgres_operator import PostgresOperator
+from airflow.operators.python_operator import PythonOperator
+from importscripts.import_financien_grootboek import load_from_dwh
+from pgcomparator_cdc_operator import PgComparatorCDCOperator
+from postgres_check_operator import PostgresCheckOperator
+from postgres_permissions_operator import PostgresPermissionsOperator
+from provenance_rename_operator import ProvenanceRenameOperator
+from sqlalchemy_create_object_operator import SqlAlchemyCreateObjectOperator
+from sql.financien_grootboek import SQL_DROP_TMP_TABLE
+
+from common import (
+    default_args,
+    DATAPUNT_ENVIRONMENT,
+    slack_webhook_token,
+    MessageOperator,
+)
+from common.sql import SQL_CHECK_COUNT
+
+DATASTORE_TYPE: str = (
```

Comment:
This exact definition is already present is 7 other DAGs:

- src/dags/anpr.py
- src/dags/beschermde_stadsdorpsgezichten.py
- src/dags/blackspots.py
- src/dags/beheerkaart.py
- src/dags/rioolnetwerk.py
- src/dags/varen.py
- src/dags/woningbouwplannen.py

I like to see this definition moved to the `common` module and removed from those 7 DAG and this new DAG.